modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -158,278 +153,285 @@ Deploying myMuleApp ...
 
+To edit the settings for an application:
 
-== Configuring a Deployed Application
+. In the *Applications* tab, the application name.
+. Click the *Settings* tab.
+. Make your changes and click *Apply Changes*
 
-Most of the settings you specified when deploying the application can be edited after the application is deployed to CloudHub.
+CloudHub redeploys the application with zero downtime.
 
-To edit an application's settings:
 
-. In the *Applications* tab, select an application entry and click *Manage Application*.
-+
-image::amc-manageapplication.png[AMC_ManageApplication]
-+
-. Click the *Settings* tab.
-+
-image::viewingdeployedapp.png[ViewingDeployedApp]
 
+== Deployment Progress   
 
-== Auto-Deploy a Proxy from API Manager
+CloudHub uploads your application and begins the deployment process, switching
+the view to *Logs* so that you can monitor the deployment progress.
 
-If you have registered an API in the Anypoint Platform, you can easily run it through an auto generated proxy to track its usage and implement policies. You can deploy this proxy to CloudHub without ever needing to go into the Runtime Manager section of Anypoint Platform.
+The application status indicator changes to yellow to indicate the deployment is in progress.
+When the deployment is complete, the application status indicator changes to green, a message
+appears in the status area and in the logs to indicate that the application has deployed successfully. 
 
-From a menu in the API version page, you can trigger the deployment of your proxy and set up the CloudHub application name, environment, and gateway version to use. Then you can optionally access the Runtime Manager deployment menu for this proxy and configure advanced settings.
+[NOTE]
+While you deploy an application on Mule runtime engine version 4.1.x and later, the xref:api-manager::gatekeeper.adoc#how-gatekeeper-works[Gatekeeper] mechanism works with xref:api-manager::api-auto-discovery-new-concept.adoc[API Autodiscovery] to help determine the readiness of a deployment.
+If Gatekeeper considers the application not ready in five minutes, the deployment status is failed.
 
 == Deployment Errors
 
-If an error occurs and the application cannot be deployed, the application status indicator changes to `Failed`. You are alerted in the status area that an error occurred. Check the log details for any application deployment errors. You need to correct the error, upload the application, and deploy again.
+If an error occurs and the application cannot be deployed, the application status indicator
+changes to `Failed`, and the status area indicates that an error occurred.
+The logs include details on the application deployment errors.
 
 
 == See Also
 
-* xref:cloudhub.adoc[CloudHub]
-* xref:managing-deployed-applications.adoc[Managing Deployed Applications]
+* xref:managing-cloudhub-specific-settings.adoc[Managing CloudHub Specific Settings] 
```

Comment:
I understand that correcting existing titles is probably not possible and I should likely STFU about these....
```suggestion
* xref:managing-cloudhub-specific-settings.adoc[Managing CloudHub-Specific Settings] 
```